centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -17473,3 +17473,10 @@ msgstr "Services au statut Inconnu"
 
 msgid "OK status services"
 msgstr "Services au statut OK"
+
+
+#: centreon-bam/www/modules/centreon-bam-server/react/hooks/header/topCounter
+
+msgid "B.Activities"
```

Comment:
We should name it Business Activities instead

Reply:
I think it was, but the design specs use 'B.Activities' and 'Business Activities' would create a very long item. 
the entire header is using snapshots (but not with BA, since I don't see a simple path on how to activate licenced modules in cypress...)